overlord/ifacestate/handlers.go

```diff
@@ -1135,14 +1135,17 @@ func (m *InterfaceManager) transitionConnectionsCoreMigration(st *state.State, o
 	}
 	setConns(st, conns)
 
+	// After migrating connections in state, remove them from repo so they stay in sync and we don't
```

Comment:
The `ubuntu-core` snap used to have a configuration hook and said hook had a plug for `core-support`. I think we must remove that plug connection as well.